themes/default/build/sass/style.scss

```diff
@@ -1,68 +1,78 @@
-@import '../../css/pure.src'; // import pure as sass for extending
-
 /*
-# Brand Overview
+    Imports
+    Contents: 
+
+    1.  Pure
+    2.  Brand overview
+    3.  Layout
+    4.  Typography
+    5.  UI kit
+    6.  Forms
+    7.  Components
+    8.  Shame
+*/
+
+/* 
+    1.  Pure
+	Import pure as sass for extending
+*/
 
-Define the colours and fonts used throughout the web app
+@import '../../css/pure.src';
 
-Styleguide 1.0
+/*
```

Comment:
These are used by the styleguide - They are the main heading levels, so need to look like they did before.